rust/tvm/examples/resnet/build.rs

```diff
@@ -17,16 +17,20 @@
  * under the License.
  */
 
-use anyhow::{Context, Result};
-use std::{path::Path, process::Command};
+use std::{io::Write, path::Path, process::Command};
 
-fn main() -> Result<()> {
+fn main() {
     let output = Command::new("python3")
         .arg(concat!(env!("CARGO_MANIFEST_DIR"), "/src/build_resnet.py"))
         .arg(&format!("--build-dir={}", env!("CARGO_MANIFEST_DIR")))
         .output()
-        .with_context(|| anyhow::anyhow!("failed to run python3"))?;
```

Comment:
Is it intentional here that we're reverting the `anyhow` changes from #6401?

Reply:
Good catch, I'll put it back.